[2025-10-26T17:00:01.647+0100] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-10-26T17:00:01.674+0100] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_hourly_refresh.run_dbt_models scheduled__2025-10-26T15:00:00+00:00 [queued]>
[2025-10-26T17:00:01.684+0100] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_hourly_refresh.run_dbt_models scheduled__2025-10-26T15:00:00+00:00 [queued]>
[2025-10-26T17:00:01.685+0100] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2025-10-26T17:00:01.700+0100] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): run_dbt_models> on 2025-10-26 15:00:00+00:00
[2025-10-26T17:00:01.713+0100] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=720) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-26T17:00:01.709+0100] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dbt_hourly_refresh', 'run_dbt_models', 'scheduled__2025-10-26T15:00:00+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/dbt_hourly_refresh_dag.py', '--cfg-path', '/tmp/tmp8nf4o_nh']
[2025-10-26T17:00:01.714+0100] {standard_task_runner.py:105} INFO - Job 62: Subtask run_dbt_models
[2025-10-26T17:00:01.714+0100] {standard_task_runner.py:72} INFO - Started process 721 to run task
[2025-10-26T17:00:01.780+0100] {task_command.py:467} INFO - Running <TaskInstance: dbt_hourly_refresh.run_dbt_models scheduled__2025-10-26T15:00:00+00:00 [running]> on host f229876aebe2
[2025-10-26T17:00:01.965+0100] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Aliostadi' AIRFLOW_CTX_DAG_ID='dbt_hourly_refresh' AIRFLOW_CTX_TASK_ID='run_dbt_models' AIRFLOW_CTX_EXECUTION_DATE='2025-10-26T15:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-26T15:00:00+00:00'
[2025-10-26T17:00:01.966+0100] {taskinstance.py:731} INFO - ::endgroup::
[2025-10-26T17:00:02.006+0100] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-26T17:00:02.008+0100] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/dbt_project && dbt clean && dbt run --full-refresh']
[2025-10-26T17:00:02.022+0100] {subprocess.py:86} INFO - Output:
[2025-10-26T17:00:06.280+0100] {subprocess.py:93} INFO - [0m16:00:06  Running with dbt=1.8.3
[2025-10-26T17:00:06.677+0100] {subprocess.py:93} INFO - [0m16:00:06  Checking /opt/***/dbt_project/dbt_packages/*
[2025-10-26T17:00:06.677+0100] {subprocess.py:93} INFO - [0m16:00:06  Cleaned /opt/***/dbt_project/dbt_packages/*
[2025-10-26T17:00:06.678+0100] {subprocess.py:93} INFO - [0m16:00:06  Checking /opt/***/dbt_project/target/*
[2025-10-26T17:00:06.683+0100] {subprocess.py:93} INFO - [0m16:00:06  Cleaned /opt/***/dbt_project/target/*
[2025-10-26T17:00:06.683+0100] {subprocess.py:93} INFO - [0m16:00:06  Finished cleaning all paths.
[2025-10-26T17:00:09.748+0100] {subprocess.py:93} INFO - [0m16:00:09  Running with dbt=1.8.3
[2025-10-26T17:00:10.069+0100] {subprocess.py:93} INFO - [0m16:00:10  Registered adapter: postgres=1.8.2
[2025-10-26T17:00:10.245+0100] {subprocess.py:93} INFO - [0m16:00:10  Unable to do partial parsing because saved manifest not found. Starting full parse.
[2025-10-26T17:00:12.296+0100] {subprocess.py:93} INFO - [0m16:00:12  [[33mWARNING[0m]: Deprecated functionality
[2025-10-26T17:00:12.297+0100] {subprocess.py:93} INFO - The `tests` config has been renamed to `data_tests`. Please see
[2025-10-26T17:00:12.297+0100] {subprocess.py:93} INFO - https://docs.getdbt.com/docs/build/data-tests#new-data_tests-syntax for more
[2025-10-26T17:00:12.297+0100] {subprocess.py:93} INFO - information.
[2025-10-26T17:00:12.911+0100] {subprocess.py:93} INFO - [0m16:00:12  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-10-26T17:00:12.911+0100] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2025-10-26T17:00:12.911+0100] {subprocess.py:93} INFO - - models.intermediters
[2025-10-26T17:00:13.114+0100] {subprocess.py:93} INFO - [0m16:00:13  Found 3 models, 6 data tests, 3 sources, 428 macros
[2025-10-26T17:00:13.117+0100] {subprocess.py:93} INFO - [0m16:00:13
[2025-10-26T17:00:13.270+0100] {subprocess.py:93} INFO - [0m16:00:13  Concurrency: 4 threads (target='dev')
[2025-10-26T17:00:13.271+0100] {subprocess.py:93} INFO - [0m16:00:13
[2025-10-26T17:00:13.287+0100] {subprocess.py:93} INFO - [0m16:00:13  1 of 3 START sql incremental model analytics.int_performance_station_hour ...... [RUN]
[2025-10-26T17:00:13.292+0100] {subprocess.py:93} INFO - [0m16:00:13  2 of 3 START sql incremental model analytics.int_quality_station_hour .......... [RUN]
[2025-10-26T17:00:14.912+0100] {subprocess.py:93} INFO - [0m16:00:14  1 of 3 OK created sql incremental model analytics.int_performance_station_hour . [[32mSELECT 3083[0m in 1.62s]
[2025-10-26T17:00:20.851+0100] {subprocess.py:93} INFO - [0m16:00:20  2 of 3 OK created sql incremental model analytics.int_quality_station_hour ..... [[32mSELECT 3083[0m in 7.56s]
[2025-10-26T17:00:20.854+0100] {subprocess.py:93} INFO - [0m16:00:20  3 of 3 START sql incremental model analytics.int_quality_linewise_datehour ..... [RUN]
[2025-10-26T17:00:21.189+0100] {subprocess.py:93} INFO - [0m16:00:21  3 of 3 OK created sql incremental model analytics.int_quality_linewise_datehour  [[32mSELECT 395[0m in 0.33s]
[2025-10-26T17:00:21.208+0100] {subprocess.py:93} INFO - [0m16:00:21
[2025-10-26T17:00:21.208+0100] {subprocess.py:93} INFO - [0m16:00:21  Finished running 3 incremental models in 0 hours 0 minutes and 8.09 seconds (8.09s).
[2025-10-26T17:00:21.243+0100] {subprocess.py:93} INFO - [0m16:00:21
[2025-10-26T17:00:21.244+0100] {subprocess.py:93} INFO - [0m16:00:21  [32mCompleted successfully[0m
[2025-10-26T17:00:21.244+0100] {subprocess.py:93} INFO - [0m16:00:21
[2025-10-26T17:00:21.245+0100] {subprocess.py:93} INFO - [0m16:00:21  Done. PASS=3 WARN=0 ERROR=0 SKIP=0 TOTAL=3
[2025-10-26T17:00:22.113+0100] {subprocess.py:97} INFO - Command exited with return code 0
[2025-10-26T17:00:22.147+0100] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-10-26T17:00:22.147+0100] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dbt_hourly_refresh, task_id=run_dbt_models, run_id=scheduled__2025-10-26T15:00:00+00:00, execution_date=20251026T150000, start_date=20251026T160001, end_date=20251026T160022
[2025-10-26T17:00:22.199+0100] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-10-26T17:00:22.217+0100] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-26T17:00:22.219+0100] {local_task_job_runner.py:245} INFO - ::endgroup::
